src/app/components/core/system-log/system-log.component.ts

```diff
@@ -66,24 +66,33 @@ export class SystemLogComponent implements OnInit, OnDestroy {
           // processName north_C represents Northbound services
           const north_C = [];
           const north = [];
+          const management = [];
+          const dispatcher_c = [];
+
           data.schedules.forEach(sch => {
-            if ('south_c'.includes(sch.processName)) {
+            if ('south_c' === sch.processName) {
               south_c.push(sch);
             }
-            if ('notification_c'.includes(sch.processName)) {
+            if ('notification_c' === sch.processName) {
               notification_c.push(sch);
             }
-            if ('north_c'.includes(sch.processName)) {
+            if ('north_c' === sch.processName) {
               north_c.push(sch);
             }
-            if ('north_C'.includes(sch.processName)) {
+            if ('north_C' === sch.processName) {
               north_C.push(sch);
             }
-            if ('north'.includes(sch.processName)) {
+            if ('north' === sch.processName) {
               north.push(sch);
             }
+            if ('management' === sch.processName) {
+              management.push(sch);
```

Comment:
ensure this shows captured logs of child process of this task manage / poll schedule

Reply:
Under Logs Task section. each timed / interval based task should also appear for poll schedule.